lib/streams/json-parse-stream.js

```diff
@@ -0,0 +1,24 @@
+var Parser = require('jsonparse');
+var ZTransform = require('../transform');
+var util = require('util');
+
+function JSONParseStream (options) {
```

Comment:
It looks like this stream isn't actually parsing JSON in a streaming fashion - it looks like it's parsing objects, one chunk at a time.   This is still useful (for example, to parse a file that is newline-separated JSON, you could pipe the stream through a SplitStream then this stream), but probably shouldn't be called JSONParseStream .  Perhaps rename it to JSONObjectParseStream ?